lib/roku-develop.coffee

```diff
@@ -245,6 +248,57 @@ module.exports        = RokuDevelop =
   toggle: ->
     if @panel.isVisible() then @panel.hide() else @panel.show()
 
+
+  #
+  # Invoked when the roku-develop:switch-files command is issued
+  #
+  switch: ->
```

Comment:
If you remove your old 'switch' code, I'll go ahead and merge the PR into the master branch, and release later today.

Reply:
oh, my fault—I checked it out locally, then hit "commit suggestion" and didn't realize that didn't remove the old code. it's done.

Reply:
Version 0.1.12 has now been released to the Atom package repository. Thanks for your contribution. Let me know if you have any issues with the code.